coretex/cli/modules/ui.py

```diff
@@ -77,6 +77,17 @@ def previewConfig(config: Dict[str, Any]) -> None:
     stdEcho(tabulate(table))
 
 
+def outputUrl(config: Dict[str, Any], entity: str, id: int) -> None:
+    if entity == "model":
+        stdEcho(f"{config.get('serverUrl')}model-item?id={id}")
```

Comment:
Create a function which generates the right part of the URL inside of `NetworkObject`. Override as needed in child classes for entities to return desired values.